### ISSUE TITLE
apt-get update unable to find packages

### ISSUE THREAD

**Issue by kudryk** [confluentinc/cp-docker-images#764]
My project is using kafka-connect, and we have slightly customized kafka-connect. Using the `confluentinc/cp-kafka-connect:5.3.0` docker image as the base for our image (upgrading from 5.2.1 which built without error), when `apt-get update` is executed, the following error is occurs:

```
...
Get:13 https://s3-us-west-2.amazonaws.com stable/main amd64 Packages
Err https://s3-us-west-2.amazonaws.com stable/main amd64 Packages
  HttpError404
Get:14 http://security.debian.org jessie/updates/main amd64 Packages [870 kB]
Get:15 http://archive.debian.org jessie Release [148 kB]
Get:16 http://archive.debian.org jessie/main amd64 Packages [9098 kB]
Fetched 10.2 MB in 19s (525 kB/s)
W: Failed to fetch https://s3-us-west-2.amazonaws.com/staging-confluent-packages-5.3.0/deb/5.3/dists/stable/main/binary-amd64/Packages  HttpError404

E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100
```

Looking at `/etc/apt/sources.list` in the image I see it contains the following entry:
```
deb [arch=amd64] https://s3-us-west-2.amazonaws.com/staging-confluent-packages-5.3.0/deb/5.3 stable main
```
I've tried to access that bucket and I only get "The specified bucket does not exist" error message.

Any suggestion on how to get around this? Is that entry correct?

**Comment by ewencp** [comment]
Interesting issue -- this is due to the images being built against a staged set of packages before we officially release (so the final URL that *should* be there isn't valid yet at the time).

You can replace the url in `/etc/apt/sources.list `with `https://packages.confluent.io/deb/5.3` (basically just make sure it matches what's described in the deb installation docs: https://docs.confluent.io/current/installation/installing_cp/deb-ubuntu.html#get-the-software).

**Comment by nick-zh** [comment]
@ewencp any idea if this will be fixed? i get this on the `cp-kafka:5.3.0` as well...

**Comment by benesch** [comment]
You can work around it for now by adjusting the URL before running `apt-get update`:

```Dockerfile
FROM confluentinc/cp-enterprise-kafka:5.3.0

# https://github.com/confluentinc/cp-docker-images/issues/764
RUN sed -i s,https://s3-us-west-2.amazonaws.com/staging-confluent-packages-5.3.0/deb/5.3,https://packages.confluent.io/deb/5.3, /etc/apt/sources.list

RUN apt-get update && apt-get -qy install PACKAGES
```
